src/Tools/dotnet-monitor/ConfigurationTokenParser.cs

```diff
@@ -34,16 +38,22 @@ internal sealed class ConfigurationTokenParser
         public const string SubstitutionSuffix = ")";
         public const string Separator = ".";
 
+        private const string MonitorInfoReference = "Monitor";
         private const string ProcessInfoReference = "Process";
+
         private const string RuntimeId = "RuntimeId";
         private const string ProcessId = "ProcessId";
         private const string ProcessName = "Name";
         private const string CommandLine = "CommandLine";
+        private const string HostName = "HostName";
```

Comment:
I went with the PascalCase `"HostName"` instead of `"Hostname"` based on the .NET APIs usage like `Dns.GetHostName`